File: elza/elza-core/src/main/java/cz/tacr/elza/controller/vo/ApAttributesInfoVO.java
package cz.tacr.elza.controller.vo;

import java.util.ArrayList;
import java.util.List;

public class ApAttributesInfoVO {

    /**
     * Vyhodnocené typy a specifikace atributů, které jsou třeba pro založení přístupového bodu
     */
    private List<ApCreateTypeVO> attributes = new ArrayList<>();

    /**
     * Chyby při validaci
     */
    private List<String> errors = null;

    public List<ApCreateTypeVO> getAttributes() {
        return attributes;
    }

    public void setAttributes(List<ApCreateTypeVO> attributes) {
        this.attributes = attributes;
    }

    public List<String> getErrors() {
        return errors;
    }

    public void setErrors(List<String> errors) {
        this.errors = errors;
    }
}

File: elza/elza-react/src/api/ApAttributesInfoVO.ts
import {ApCreateTypeVO} from "./ApCreateTypeVO";

export interface ApAttributesInfoVO {

    /**
     * Vyhodnocené typy a specifikace atributů, které jsou třeba pro založení přístupového bodu
     */
    attributes: ApCreateTypeVO[];

    /**
     * Chyby při validaci
     */
    errors?: string[];
}
